lxd/events/common.go

```diff
@@ -126,17 +65,8 @@ func (e *listenerCommon) Close() {
 		return
 	}
 
-	logger.Debug("Event listener server handler stopped", logger.Ctx{"listener": e.ID(), "local": e.Conn.LocalAddr(), "remote": e.Conn.RemoteAddr()})
+	logger.Debug("Event listener server handler stopped", logger.Ctx{"listener": e.ID(), "local": e.EventListenerConnection.LocalAddr(), "remote": e.EventListenerConnection.RemoteAddr()})
```

Comment:
As we are embedding `EventListenerConnection` directly into `listenerCommon` it should be possible to just use `e.LocalAddr()` here.